api/src/opentrons/protocol_engine/state/commands.py

```diff
@@ -203,6 +203,9 @@ class CommandState:
     This value can be used to generate future hashes.
     """
 
+    failed_command_errors: List[ErrorOccurrence]
+    """List of errors that occurred during run execution."""
```

Comment:
```suggestion
    """List of command errors that occurred during run execution."""
```

Because it excludes the fatal run-level error, e.g. if the user's script raised an `IndexError`.